### DISCUSSION TITLE
Gorgeous!

### DISCUSSION THREAD

**Discussion by adrian5** [2giosangmitom/nightfall.nvim#52] [General]
Hello Chien, just wanted to tell you what a great looking high contrast theme you created! I've been working on my own derivation of Dracula, but I think I might just use this instead. Love the hues and how very polished the theme is. Keep up the good work, I'll try to contribute if I see anything that needs attention but it doesn't look like it.

**Comment by 2giosangmitom** [comment]
Hello! I'm thrilled you like it. You're more than welcome to contribute to this plugin!
